src/Components/MainTableView/DeleteTuple/DeleteTuple.tsx

```diff
@@ -90,11 +90,21 @@ export default class DeleteTuple extends React.Component<DeleteTupleProps, Delet
     // set status true for deleting entry, switch to false once api responds
     this.props.deleteInAction(true)
 
+    // Covert the tuple to delete to restriction
+    var restrictionsInAPIFormat: Array<any> = [];
+    for (const [key, value] of Object.entries(tupleToDelete)) {
+      restrictionsInAPIFormat.push({
+        attributeName: key,
+        operation: '=',
+        value: String(value)
+      })
+    }
+    console.log(restrictionsInAPIFormat)
```

Comment:
This looks like a debug print that should likely be removed.

Reply:
Fixed